### ISSUE TITLE
Set body content at 74.75% of container size for accessibility.  

### ISSUE THREAD

**Issue by bspeare** [uiowa/uiowa01#188]
# The Problem

Pages without a sidebar have character lengths that violate accessibility best practices. 

![Screen Shot 2019-11-07 at 9 08 01 AM](https://user-images.githubusercontent.com/1036433/68400734-1fb86600-013e-11ea-8057-0b023214e512.png)

# Proposed Solution

For basic content pages and pages that use the one-column layout for a section, width is set at 73% of the container.  

![Screen Shot 2019-11-07 at 9 01 39 AM](https://user-images.githubusercontent.com/1036433/68400947-83429380-013e-11ea-86ff-dddc95aecd55.png)

**Comment by pyrello** [comment]
Can you provide some additional context as to how you arrived at the 74.75% width for a single column of basic text? Is this via an accessibility tool? You can safely assume that I don't know as much about this subject as I should.

**Comment by bspeare** [comment]
74% is just a starting point that I chose.  We will probably need to reduce it to more of a 70/30 ratio to have a good line character length.  https://www.w3.org/WAI/tutorials/page-structure/styling/#line-length

**Comment by pyrello** [comment]
Is this change actually necessary? Based on the link above:

> Another way to ensure good line-length is to use a fluid layout or apply responsive web design techniques that allow the user to resize the window to find a comfortable line length.

I think that if we are going to apply this, it should only be applied at screen widths above a certain point, as it seems like it shouldn't be applied to phone screen widths.

<img width="507" alt="Screen Shot 2019-11-11 at 2 54 53 PM" src="https://user-images.githubusercontent.com/709009/68620604-76ea6d80-0493-11ea-859e-442df04d4bed.png">

**Comment by bspeare** [comment]
Correct, I just need to adjust the breakpoint for the width.
